packages/firestore/src/lite-api/query.ts

```diff
@@ -168,10 +168,12 @@ export function query<T>(
 
   if (nonFilters !== undefined) {
     queryConstraints = queryConstraints.concat(
-      nonFilters as QueryNonFilterConstraint[]
+      nonFilters as AppliableConstraint[]
```

Comment:
I realize it's not a change made in this PR, but I just noticed it:
it looks like there's a bug on line 166 above (the return value of concat is ignored)?